sign_oca/views/res_partner_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" ?>
+<odoo>
+
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form (in sign_oca)</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button
+                    class="oe_stat_button"
+                    type="object"
+                    name="action_show_signer_ids"
+                    attrs="{'invisible': [('signer_count', '=', 0)]}"
+                    icon="fa-pencil"
+                    groups="sign_oca.sign_oca_group_user"
```

Comment:
For this version, the groups should be put at view level, not element level, or you will have an error.

Reply:
I'm not sure if I understand what you mean. In version 14, groups can be added to buttons to not be shown. I have tested it and it works fine.

Reply:
As said, if you enter in this view without having the permissions, it will give an access error. Please put the group at view level, not element level. Something like:

```xml
    <record id="view_partner_form" model="ir.ui.view">
        <field name="name">res.partner.form (in sign_oca)</field>
        <field name="model">res.partner</field>
        <field name="inherit_id" ref="base.view_partner_form" />
        <field name="groups_id" eval="[(4, ref('...')]" />
        <field name="arch" type="xml">
            <div name="button_box" position="inside">
```

Reply:
I do not agree that it would give access error. However, I change it since it seems right to me anyway.

Reply:
If there are no access errors, it's because this model is readable for all users, but it's a good practice to do it this way because:

- It avoids needing to access the model without need.
- It avoids access errors when the model is not readable.